src/main/java/io/confluent/idesidecar/restapi/resources/RestProxyResource.java

```diff
@@ -47,18 +46,24 @@ public class RestProxyResource {
   static final MultiMap NO_HEADERS = MultiMap.caseInsensitiveMultiMap();
   static final Map<String, String> NO_PATH_PARAMS = Map.of();
 
-  // Add this static declaration alongside the other static constants
-  static final String ccloudApiProxyRegex = ConfigProvider
+  static final String ccloudApiControlPlaneProxyRegex = ConfigProvider
```

Comment:
Per convention, we should only use upper case characters for constant names (with words being separated by an underscore):

```suggestion
  static final String CCLOUD_API_CONTROL_PLANE_PROXY_REGEX = ConfigProvider
```